plrust/src/tests.rs

```diff
@@ -1292,3 +1292,29 @@ tokio = { version = "1.19.2", features = ["rt", "net"]}"#
         ]
     }
 }
+
+#[test]
+#[cfg(feature = "trusted")]
```

Comment:
don't need to feature flag this.  If we merge this, it should happen regardless of trusted mode -- `plrustc` is used in both cases.

Reply:
Oh, is it?

Reply:
Yeah.  We apply the lints no matter what.  `trusted` really just means "use postgrestd".